src/Playwright/Transport/Protocol/Generated/RequestInitializer.cs

```diff
@@ -36,7 +36,7 @@ internal class RequestInitializer
 
         public string Method { get; set; }
 
-        public string PostData { get; set; }
+        public byte[] PostData { get; set; }
```

Comment:
These are generated, so we can't change them since over the wire it's transferred as a base64 string. The rest looks good.

Reply:
System.Text.Json will automatically convert byte[] properties to base64. https://dotnetfiddle.net/i5Ae28
But if this got generated I will revert it.